pkg/network/protocols/events/consumer_test.go

```diff
@@ -37,6 +38,7 @@ func TestConsumer(t *testing.T) {
 	}
 
 	const numEvents = 100
+	_, _ = sysconfig.New("/doesnotexist")
```

Comment:
> replace direct access to ddconfig.SystemProbe

This would be even more code change I think. I'm exploring a different way to manage this, so you cannot make a mistake.